hugegraph-api/src/main/java/com/baidu/hugegraph/auth/HugeGraphAuthProxy.java

```diff
@@ -100,6 +106,8 @@
     }
 
     private static final Logger LOG = Log.logger(HugeGraphAuthProxy.class);
+    private final Cache<Id, RateLimiter> auditPerUserRates;
+    private double logRate;
```

Comment:
set final

Reply:
The rate may be adjusted dynamically in future, now shall we set to final?

Reply:
we can remove the final if needed